base_controller_lib/build.gradle

```diff
@@ -14,17 +14,26 @@
  * the License.
  */
 
+// UsbSerialLibrary is not yet available on Maven, so the path to the artifact
+// has to be sourced to work properly with tools like AndroidStudio.
 dependencies {
-    compile 'org.ros.android_core:android_10:[0.2, 0.3)'
-    compile 'com.hoho.android.usbserial:UsbSerialLibrary:[0.1.1, 0.2)'
+    compile 'org.ros.rosjava_core:rosjava:[0.3.0]'
```

Comment:
Hmm ... I'm thinking that if we release updates that modify API and would break applications that use it, we should release it under a new major number. So for this case what it means is that, if releasing the NativeNodeMain changes would break apps like this, we should release it as rosjava_core 0.4.0 not 0.3.x. As a result, this client that uses it should be able to require the entire range, not a specific version.
Requiring a specific minor version is a very rare thing, only used for very specific, temporary, short-lived hacks.